core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1024,6 +1024,7 @@ protected void openFullScreen() {
     toolbarContainer.setVisibility(View.GONE);
     bottomToolbar.setVisibility(View.GONE);
     exitFullscreenButton.setVisibility(View.VISIBLE);
+    exitFullscreenButton.getBackground().setAlpha(26);
```

Comment:
The exact 60% of 255 made the button hardly visible to an extent, so i increased the number by 10 and this looks better

Reply:
60% of 255 is 153, I am not sure where 26+/-10 comes into this